File: verifier-core/src/main/java/com/jetbrains/pluginverifier/verifiers/method/MethodVerifier.java
package com.jetbrains.pluginverifier.verifiers.method;

import com.intellij.structure.resolvers.Resolver;
import com.jetbrains.pluginverifier.api.VContext;
import org.objectweb.asm.tree.ClassNode;
import org.objectweb.asm.tree.MethodNode;

/**
 * @author Dennis.Ushakov
 */
public interface MethodVerifier {
  void verify(ClassNode clazz, MethodNode method, Resolver resolver, VContext ctx);
}

File: verifier-core/src/main/java/com/jetbrains/pluginverifier/verifiers/method/MethodVerifier.kt
package com.jetbrains.pluginverifier.verifiers.method

import com.jetbrains.pluginverifier.verifiers.VerificationContext
import com.jetbrains.pluginverifier.verifiers.resolution.Method

interface MethodVerifier {
  fun verify(method: Method, context: VerificationContext)
}
